src/context/context.service.ts

```diff
@@ -94,8 +94,24 @@ export class ContextService {
     return $('<div>').html($.html()).text().trim();
   }
 
-  setHtmlBody(body: string): void {
-    this.cheerioBody = cheerio.load(body);
+     setHtmlBody(
+    body: string,
+    loadAsDocument = false,
+    divId = 'konviw-ontent',
```

Comment:
I would not make the divId configurable because there are too much dependencies on the code and stylesheets on this one. So better to keep it by convention as #Content.

Reply:
Ok, makes sense and removes a bit of complexity in the implementation - an embedding app can put it inside another div and it will work fine.